### ISSUE TITLE
response checks

### ISSUE THREAD

**Issue by joprice** [toolebox-io/gatling-lambda-extension#16]
I cannot find any checks for validating the response. Is something similar to the http check needed?

**Comment by Tyrannoseanus** [comment]
Yeah so far I've only implemented the invoke part of the protocol. If we want to validate the response (Which is a great idea, and provides good feature parity with the HTTP protocol) then something like the HTTP checks would need to be added.

**Comment by joprice** [comment]
I could take a stab at it, but I won't have time for a week or so.

**Comment by kieranbop** [comment]
How did you get on @joprice ? I will give it a whirl if you've had no luck yet.

**Comment by joprice** [comment]
I haven't had the time, and was able to gather the information I needed with just the latency data for now. Go for it!

**Comment by hardiksoni13** [comment]
I am looking into this now, hopefully we will have this soon
